src/main/java/jsr223/docker/compose/utils/DockerComposePropertyLoader.java

```diff
@@ -45,30 +46,25 @@
 @Log4j
 public class DockerComposePropertyLoader {
 
-    @lombok.Getter
-    public String dockerHost;
-
-    @lombok.Getter
-    public String dockerComposeCommand;
-
-    @lombok.Getter
-    public String sudoCommand;
-
-    @lombok.Getter
-    public boolean useSudo;
-
-    protected Properties properties;
-
-    protected String configurationFile = "config/scriptengines/docker-compose.properties";
-
-    private DockerComposePropertyLoader()  {
+    private final static String CONFIGURATION_FILE = "config/scriptengines/docker-compose.properties";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)